onlyOffice/src/test/java/org/exoplatform/platform/qa/ui/onlyOffice/OnlyOfficeTestIT.java

```diff
@@ -0,0 +1,137 @@
+import static org.exoplatform.platform.qa.ui.selenium.locator.HomePageLocator.ELEMENT_BAR_PROGRESS;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.ELEMENT_SKIP_BUTTON;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ActivityStreamLocator.ELEMENT_CLOSE_DOCUMENT_PREVIEW1;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ActivityStreamLocator.*;
+
+
+public class OnlyOfficeTestIT extends Base {
+
+  HomePagePlatform            homePagePlatform;
+  NavigationToolbar           navigationToolbar;
+  SiteExplorerHome            siteExplorerHome;
+  ManageLogInOut              manageLogInOut;
+  ConnectionsManagement       connectionsManagement;
+  ActivityStream              activityStream;
+  SpaceManagement             spaceManagement;
+  TestBase                    testBase;
+
+
+  @BeforeEach
+  public void setupBeforeMethod() {
+
+    homePagePlatform            =new HomePagePlatform(this);
+    navigationToolbar           =new NavigationToolbar(this);
+    siteExplorerHome =new SiteExplorerHome(this);
+    manageLogInOut = new ManageLogInOut(this);
+    //manageLogInOut.signInCas(PLFData.DATA_USER1, "gtngtn");
+    spaceManagement             =new SpaceManagement(this);
+
+    activityStream      =new ActivityStream(this);
+    manageLogInOut.signInCas("root", "gtn");
+    connectionsManagement = new ConnectionsManagement(this);
+    if ($(ELEMENT_SKIP_BUTTON).is(Condition.exist)) {
+      $(ELEMENT_SKIP_BUTTON).click();
+    }
+  }
+
+  @Test
+  public void EditOnline_BTN_US01 () {
+    String Document = "OO_test";
+    info("Upload document");
+    homePagePlatform.goToDocuments();
+    siteExplorerHome.uploadFile(Document + ".docx");
+    info("Open document");
+    siteExplorerHome.checkButtonDocument(Document);
+    siteExplorerHome.clickViewDocumentButton();
+    switchTo().window(1);
+    siteExplorerHome.clickEditOnine();
+    switchTo().window(2);
+    switchTo().frame(0);
+    $(byText(Document + ".docx")).waitUntil(Condition.visible,Configuration.openBrowserTimeoutMs);
+    switchTo().window(2).close();
+    switchTo().window(1).close();
+    switchTo().window(0);
+    info("Delete document");
+    siteExplorerHome.checkButtonDocument(Document);
+    siteExplorerHome.clickDeleteButtonDocument();
+    manageLogInOut.signOut();
+  }
+
+
+  @Test
+  public void EditOnline_BTN_US02 () {
+    String Document = "OO_test.docx";
+    String space_test = "space_1_"+getRandomNumber();
+    info("Edit Online from document from AS");
+    activityStream.UploadFileFromAS(Document);
+    homePagePlatform.refreshUntil($(byXpath(ELEMENT_CHECK_FILE_IS_UPLOADED.replace("$doc",Document))),Condition.visible,Configuration.timeout);
+    ELEMENT_EDIT_ONLINE_BUTTON_FROM_AS.waitUntil(Condition.visible,Configuration.openBrowserTimeoutMs).click();
```

Comment:
Add static import of Condition and Configuration to avoid repeating it everywhere.